content/html/concepts/semantic-html/terms/summary/summary.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '<summary>'
+Description: 'Used as a label for the <details> element and when clicked reveals additional information.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<summary>` semantic HTML element is used as a label for the `<details>` element which is visible no matter what state the `<details>` element is in.
```

Comment:
I wonder if this introductory sentence could be revised or split into 2 smaller sentences?